### DISCUSSION TITLE
Flet and Google Colab

### DISCUSSION THREAD

**Discussion by tellts** [flet-dev/flet#5500] [Q&A]
Please tell me in detail, step by step, with example commands and example code, using the information from these two articles, how to use Flat in Google Colab.
https://flet.dev/docs/guides/python/deploying-web-app/hosting-providers/self-hosting

https://saturncloud.io/blog/is-there-a-general-way-to-run-web-applications-on-google-colab/

import flet as ft

def main(page: ft.Page):
    t = ft.Text(value="Hello, world!", color="green")
    page.controls.append(t)
    page.update()

ft.app(target=main, view=ft.AppView.WEB_BROWSER)

**Comment by sharsha315** [comment]
Hi, is this issue still open. Please, can I have the opportunity to work on this issue.

**Comment by ndonkoHenri** [comment]
Try this? https://github.com/flet-dev/flet/discussions/2893#discussion-6381813

**Comment by tellts** [comment]
Hello. Yes, I still need help.

**Comment by dpaes** [comment]
up
